prism/src/main/java/org/apache/falcon/resource/proxy/SchedulableEntityManagerProxy.java

```diff
@@ -310,6 +319,12 @@ public APIResult update(
             @Dimension("colo") @QueryParam("colo") String ignore,
             @QueryParam("skipDryRun") final Boolean skipDryRun) {
 
+        try {
+            isEntityPartOfAnExtension(EntityUtil.getEntity(type, entityName));
```

Comment:
The new definition can/might have removed the tag information, hence old entity needs to be used.